toe/src/urbit/sur/toe.hoon

```diff
@@ -0,0 +1,32 @@
+::  /sur/toe/hoon
+::
+|%
+::
++|  %data-structures
+::
++$  game-room    [board=board-game =toers who=@p]
++$  game-rooms   (list [=ship room=(unit game-room)])
++$  grid         (list [%klr styx])
```

Comment:
This seems a bit suspect to have in the `/sur`. Seems like this is used for _printing_ the grid, but not necessarily storing/exposing it?